app.js

```diff
@@ -1,8 +1,21 @@
 const express = require('express');
 const app = express();
-const { getAllTopics } = require('./controllers/topics.controllers')
+const { getApi, getAllTopics } = require('./controllers/topics.controllers')
 
-app.get("/api/topics", getAllTopics);
+app.use(express.json());
```

Comment:
Do we need this line of code yet? What is it doing? We don't have a test written that has driven us to write this line of code so lets remove it for now and bring it in when it's needed 👍